fabfile.py

```diff
@@ -231,21 +231,25 @@ def help(name):
 @task
 def production():
     """Select production environment"""
+    env['environment'] = 'production'
```

Comment:
We use the form `env.gateway` (rather than square brackets) in other places, should we use the same form here? My Python-fu is not strong, so open to suggestions.